tests/e2e/elasticsearch/es-multiinstance/05-delete.yaml

```diff
@@ -2,8 +2,5 @@
 # the test
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
-delete:
-- apiVersion: v1
```

Comment:
Why this doesn't work?

Reply:
Because KUTTL ony allows removing stuff from the same namespace